code/script.js

```diff
@@ -26,22 +71,98 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
+  
+// This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
+ 
+chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+// Chat starts here
+//Here you can see the questionNumbers and there order set with a number 1-3
+
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1;
+  showMessage(`Hello, what's your name?`, 'bot');
+}
+
+const usrInput = (answer) => {
+  showMessage(answer, 'user')
+}
+
+const question1 = (question) => {
+  const input = document.getElementById('user-input').value
+  questionNumber++
+  showMessage(`Okay, ${input}, what do you want!?`, 'bot')
+}
+  
+const question2 = (question) => {
+  questionNumber++
+  showMessage(`I don't care. Which one of these do you need help with?`, 'bot')
+  form.style.display='none'
+  factButtons.style.display='flex'
 }
 
-// Set up your eventlisteners here
+//Here we set a setTimeout for each answer - in comparison with the bots question, which are all set in the function above. 
+
+const mother = () => {
+    setTimeout (() => showMessage(`Mother in law`, 'user'), 1000)
+    setTimeout (() => showMessage(`🔪+💣+💊=🪦 Anything else?`, 'bot'), 2000)
+  form.style.display='flex'
+  factButtons.style.display='none'
+}
+
+const trauma = () => {
+  setTimeout(() => showMessage(`Past trauma?`, 'user'), 1000)
+  setTimeout(() => showMessage(`Have some cake! Anything else?`, 'bot'), 2000)
+  form.style.display='flex'
+  factButtons.style.display='none'
+}
+
+const friends = () => {
+  setTimeout(() => showMessage(`Making friends`, 'user'), 1000)
+  setTimeout (() => showMessage(`<a href="https://www.youtube.com/watch?v=WCSQwPv6Z-k">Watch me.</a><br>Anything else?`, 'bot'), 2000)
+  form.style.display='flex'
+  factButtons.style.display='none'
+}
+    
+const question3 = (question) => {
+  questionNumber++
+  showMessage(`I'm hungry can you wait 30min?`, 'bot')
+  form.style.display='none'
+  yesNoButtons.style.display='flex'
+}
+
+
+//Here is out conditional. Depending on if the user answers yes or no.
+
+const yesNoAnswer = (type) => {
+  
+  if (type === 'yes') {
+   setTimeout (() => showMessage(`Yes.`, 'user'), 500)
+   setTimeout (() => showMessage(`Great, brb!`, 'bot'), 2000)
+   setTimeout (() => showMessage(`Watch <a href="https://www.youtube.com/watch?v=pnr3_h06kz8">this</a> while you wait.`, 'bot'), 3000)
+   setTimeout (() => yesNoButtons.style.display='none', 3000)
+   }
+ else {
+   setTimeout (() => showMessage(`No..`, 'user'), 500)
+   setTimeout (() => showMessage(`Dude.. rude! Try again.`, 'bot'), 2000)
+   setTimeout (() => yesButton.style.display='flex', 2000)
```

Comment:
Fun addition that you only show a yes-button if the user answers no the first time!